frontend/.eslintrc

```diff
@@ -63,7 +63,7 @@
           "ecmaFeatures": { "jsx": true },
           "ecmaVersion": 2020,
           "sourceType": "module",
-          "project": "./tsconfig.json"
+          "project": "**/tsconfig.json"
```

Comment:
I don't think this works. `**/foo.json` would also get rid of the import errors...but it just disables eslint.